crates/router/src/routes/app.rs

```diff
@@ -1515,6 +1515,20 @@ impl Webhooks {
     }
 }
 
+pub struct RelayWebhooks;
+
+#[cfg(feature = "oltp")]
+impl RelayWebhooks {
+    pub fn server(state: AppState) -> Scope {
+        use api_models::webhooks as webhook_type;
+        web::scope("/webhooks/relay")
```

Comment:
Have we confirmed that this doesn't affect the existing incoming webhooks flows, due to the possibly conflicting routes?